packit/schema.py

```diff
@@ -52,7 +52,8 @@
         "actions": {
             "type": "object",
             "properties": {
-                a: {"type": "string"} for a in ActionName.get_possible_values()
+                a: {"type": "array", "items": {"type": "string"}}
```

Comment:
This means that we would no longer support action as a string, only as a list of strings, which means that we would break compatibility which would be really frustrating for our users.

Please make it work that the old way, providing a string for an action, would still be acceptable.